pylua/test/websocket_test1.lua

```diff
@@ -0,0 +1,17 @@
+require('pylua/network')
+
+function lua_test1()
```

Comment:
@OHerasym, rename file, rename test, don't use print. Add auto validation: send predefined string, server echo it, assert the reply.